templates/web/base/admin/user_import.html

```diff
@@ -0,0 +1,76 @@
+[% INCLUDE 'admin/header.html' title=loc("User Import") -%]
+[% PROCESS 'admin/report_blocks.html' %]
+
+[% status_message %]
+
+<form method="post" id="user_edit" action="[% c.uri_for( 'user_import' ) %]" enctype="multipart/form-data" accept-charset="utf-8">
+    <input type="hidden" name="token" value="[% csrf_token %]" >
+    <input type="hidden" name="submit" value="1" >
+
+    <p>
+        <label>
+            [% loc('CSV File') %]
+            <input type="file" name="csvfile" id="form_csvfile" />
+        </label>
+      <input type="submit" class="btn" name="Import users" value="[% loc('Import users') %]" />
+    </p>
+</form>
+
+[% IF new_users %]
+    <h2>[% tprintf(loc('Created %d new users'), new_users.size ) %]</h2>
+    <table>
+        <tr>
+            <th>[% loc('Name') %]</th>
+            <th>[% loc('Email') %]</th>
+            <th>[% loc('Body') %]</th>
+        </tr>
+        [% FOREACH user IN new_users %]
+            <tr>
+                <td>
+                    <a href="[% c.uri_for( c.controller('Admin').action_for('user_add'), user.id ) %]">
```

Comment:
Shouldn't this be `user_edit`, not add? :) In fact, `[% c.uri_for_action( 'admin/user_edit', user.id %]` is all you need, I think.